test/pizza-guy.spec.js

```diff
@@ -64,4 +79,40 @@ describe('pizza-guy', function() {
       expect(pizzaGuy.deliver([]).start()).toBe(undefined);
     });
   });
+
+  describe('physical files', function() {
+    it('should call onComplete after all the images where downloaded', function(done) {
+      this.timeout(4000);
```

Comment:
@alejandronanez I think that what you're describing is a different behaviour than the intended. My understanding is that @baldore wants to change the default timeout value for that specific function do be the double, and what you are describing is the setTimeout with zero to push the execution to the next event loop tick, so even if we do it, we are probably going to get the same thing.